conf.d/gitnow.fish

```diff
@@ -552,15 +557,15 @@ function tag -d "Gitnow: Tag commits following Semver"
     end
 
     # List all tags in a lexicographic order and treating tag names as versions
-    if test -z $argv
+    if test -z $argv[1]
```

Comment:
The original version throws:

> test: unexpected argument at index 3: '-a'
~/.config/fish/conf.d/gitnow.fish (line 555): 
    if test -z $argv
       ^
in function 'tag' with arguments '-z -a'
First patch tag "v0.1.0" was created.

With `$argv[1]` (so testing if the first arg exists) there is no problem, i do not understand why yet. I know too little fish scripting yet.

Reply:
Ok, but strange that this fails because `if test -z $argv` basically checks if the `tag` command was executed without arguments in order to list all tags (`__gitnow_get_tags_ordered ` function).

Reply:
Can you share how are you calling the `tag` command?

Reply:
`tag -z -a`

Reply:
Ok, I understand.
Just wrap please `$argv` around double quotes.

```suggestion
    if test -z "$argv"
```